src/index.html

```diff
@@ -11,6 +11,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="/"
+      method="get"
+      class="form-big form-style"
```

Comment:
no needs of additional word style. Apply changes everywhere
```suggestion
      class="form-big form"
```